Sources/Testing/Polling/Polling.swift

```diff
@@ -0,0 +1,463 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+/// Default values for polling confirmations.
+@available(_clockAPI, *)
+internal let defaultPollingConfiguration = (
```

Comment:
Can this be `private`?

```suggestion
private let _defaultPollingConfiguration = (
```